File: Basics of Computing and Programming/chartype.py
def main():
    c = input("Enter a character: ")
    if 'a' <= c <= 'z':
        print(f"{c} is a lower case letter.")
    elif 'A' <= c <= 'Z':
        print(f"{c} is an upper case letter.")
    elif '0' <= c <= '9':
        print(f"{c} is a digit.")
    else:
        print(f"{c} is a non-alphanumeric character.")
    
if __name__ == "__main__":
    main()

File: Advanced Programming in C++/chartype.cpp
#include<bits/stdc++.h>
using namespace std;
int main() {
    char c;
    // define the parament

    cout << "Enter a character: ";
    cin >> c;
    // input c

    if (c >= '0' && c <= '9') {
        cout << c << " is a digit." << endl;
    }
    else if (c >= 'a' && c <= 'z') {
        cout << c << " is a lower case letter." << endl;
    }
    else if (c >= 'A' && c <= 'Z') {
        cout << c << " is an upper case letter." << endl;
    }
    else {
        cout << c << " is a non-alphanumeric character." << endl;
    }
    // output ans
    return 0;
}